tests/perf/test_classification.py

```diff
@@ -258,3 +261,121 @@ def test_perf(
             criteria=self.BENCHMARK_CRITERIA,
             resume_from=fxt_resume_from,
         )
+
+
+class TestPerfSemiSLMultiClass(PerfTestBase):
+    """Benchmark single-label classification for Semi-SL task."""
+
+    MODEL_TEST_CASES = [  # noqa: RUF012
+        Benchmark.Model(task="classification/multi_class_cls", name="efficientnet_b0_semisl", category="speed"),
+        Benchmark.Model(task="classification/multi_class_cls", name="mobilenet_v3_large_semisl", category="speed"),
+        Benchmark.Model(task="classification/multi_class_cls", name="efficientnet_v2_semisl", category="accuracy"),
+        Benchmark.Model(task="classification/multi_class_cls", name="deit_tiny_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="dino_v2_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="tv_efficientnet_b3_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="tv_efficientnet_v2_l_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="tv_mobilenet_v3_small_semisl", category="other"),
+    ]
+
+    DATASET_TEST_CASES = (
+        [
+            Benchmark.Dataset(
+                name=f"cifar10@{num_label}_{idx}",
+                path=Path(f"multiclass_classification/semi-sl/cifar10@{num_label}_{idx}/supervised"),
+                group="cifar10",
+                num_repeat=1,
+                unlabeled_data_path=Path(f"multiclass_classification/semi-sl/cifar10@{num_label}_{idx}/unlabel_data"),
+                extra_overrides={
+                    "train": {
+                        "data.config.train_subset.subset_name": "train_data",
+                        "data.config.val_subset.subset_name": "val_data",
+                        "data.config.test_subset.subset_name": "val_data",
+                        "deterministic": "True",
+                    },
+                },
+            )
+            for idx in (0, 1, 2)
```

Comment:
If current configuration is used, 9 cases of every each dataset is tested with every models, which can make performence test quite longer. I think it's better to divide dataset in to 3 types and each type is corresponding to single case. How do you think about it?

Reply:
I discussed this with Emily and we decided to go this way.